File: examples/src/Slides/Background/SetBackgroundToGradient.cs
using Aspose.slides.Export

/*
This project uses Automatic Package Restore feature of NuGet to resolve Aspose.Slides for .NET API reference 
when the project is build. Please check https:#docs.nuget.org/consume/nuget-faq for more information. 
If you do not wish to use NuGet, you can manually download Aspose.Slides for .NET API from http:#www.aspose.com/downloads, 
install it and then add its reference to this project. For any issues, questions or suggestions 
please feel free to contact us using http:#www.aspose.com/community/forums/default.aspx
*/

namespace Aspose.slides.Examples.CSharp.slides.Background
{
    public class SetBackgroundToGradient
    {
        public static void Run()
        {
            #ExStart:SetBackgroundToGradient
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Slides_Presentations_Background()

            # Instantiate the Presentation class that represents the presentation file
            using (Presentation pres = new Presentation(dataDir + "SetBackgroundToGradient.pptx"))
            {

                # Apply Gradiant effect to the Background
                pres.slides[0].Background.type = BackgroundType.OwnBackground
                pres.slides[0].Background.fill_format.fill_type = FillType.Gradient
                pres.slides[0].Background.FillFormat.GradientFormat.TileFlip = TileFlip.FlipBoth

                #Write the presentation to disk
                pres.save(dataDir + "ContentBG_Grad_out.pptx", slides.export.SaveFormat.PPTX)
            }
            #ExEnd:SetBackgroundToGradient
        }
    }
}

File: examples/src/slides/background/set_background_to_gradient.py
import aspose.slides as slides


def set_background_to_gradient(global_opts):
    # Instantiate the Presentation class that represents the presentation file
    with slides.Presentation() as pres:
        # Apply Gradiant effect to the Background
        pres.slides[0].background.type = slides.BackgroundType.OWN_BACKGROUND
        pres.slides[0].background.fill_format.fill_type = slides.FillType.GRADIENT
        pres.slides[0].background.fill_format.gradient_format.tile_flip = slides.TileFlip.FLIP_BOTH

        # Write the presentation to disk
        pres.save(global_opts.out_dir + "background_gradient_format_out.pptx", slides.export.SaveFormat.PPTX)
